metabrainz/templates/index/privacy.html

```diff
@@ -156,9 +156,10 @@ <h2>{{ _('MetaBrainz donors') }}</h2>
 
   <h2>{{ _('Exceptions') }}</h2>
   <p>
-    <i><b>Please note</b>: Reasonable exceptions may apply to the above policy, for example to comply with applicable laws. The MetaBrainz 
-    server administrators (about five people in all) can of course see any information on the system they want to, but to be honest 
-    we're probably not interested enough to look.</i>
+    <i><b>Please note</b>: Reasonable exceptions may apply to the above policy, for example to comply with applicable laws.
+    Also, members of the MetaBrainz team and other personnel contracted to provide services to MetaBrainz may have access
+    to our infrastructure and we can see any information we want to, but all of us are bound by non-disclosure clauses
+    in our contracts that prevent us from making any of that information available to others.</i>
```

Comment:
Sorry I feel like I'm nit picking but I found the use of "MetaBrainz team and other personnel" followed by we/us a bit confusing. Here's a minor rewrite that I feel reads a bit better.

> Also, members of the MetaBrainz team and other personnel contracted to provide services to MetaBrainz may have access to our infrastructure and could therefore view any information they wanted to. However all personnel are contractually bound by non-disclosure clauses which prevent them from making any such information available to others.


Feel free to ignore or amend.

Reply:
Accepted, thanks!